addons/escoria-core/game/core-scripts/log/esc_logger.gd

```diff
@@ -73,10 +98,16 @@ func info(string: String, args = []):
 func warning(string: String, args = []):
 	if _get_log_level() >= LOG_WARNING:
 		var argsstr = str(args) if !args.empty() else ""
-		printerr("(W)\t" + string + " \t" + argsstr)
+		_log("(W)\t" + string + " \t" + argsstr, true)
 		if ProjectSettings.get_setting("escoria/debug/terminate_on_warnings"):
-			print_stack()
+			_log(
+				ProjectSettings.get_setting("escoria/debug/crash_message") 
```

Comment:
The crash message should include the path to the log file. Like "We're sorry, but the game crashed. Please send us the log file at (...)"

I saw that you've added the log path in the message, but at this point the specific file could be provided.

Reply:
Fixed